readme.md

```diff
@@ -68,10 +68,11 @@ If you think we missed a plugin reference, please file an issue on [Github](http
 * Semantic Linkbacks (Required) - makes IndieWeb comments and mentions look better on your site.
```

Comment:
possible alternative description(?)- Interprets Webmentions and converts them to comments and reactions that you can display in your theme.